docs/README.md

```diff
@@ -111,12 +111,14 @@ repos:
     rev: master
     hooks:
       - id: commitizen
+      - id: commitizen-branch
+        stages: [push]
 ```
 
 After the configuration is added, you'll need to run
 
 ```sh
-pre-commit install --hook-type commit-msg
+pre-commit install --hook-type commit-msg pre-push
```

Comment:
Should we separate the steps of adding these hooks? e.g., `commitizen` hook might need need `pre-push`

Reply:
I clarified the relationship between the hooks and the stages by adding a table. Let me know what you think.

Reply:
It's great! Thanks @Kurt-von-Laven